src/plugins/intel_gpu/src/kernel_selector/kernel_selector_common.h

```diff
@@ -124,6 +134,10 @@ struct KernelData {
         kd.reorderInput = false;  // for KW
         kd.autoTuneIndex = -1;
         kd.can_reuse_memory = true;
+
+        for (auto& kernel : kd.kernels) {
+            kernel.skip_execution = SkipKernelExecution<T>(orgParams);
```

Comment:
Is this to be set by all kernel generation? What about for shape agnostic kernel?

Reply:
For agnostic kernels, this value will be overwritten later